http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/HttpRoutes.scala

```diff
@@ -99,14 +72,45 @@ class HttpRoutes(
     DebuggingDirectives.logRequestResult(LoggingMagnet(myLoggingFunction))
   }
 
+  implicit val myExceptionHandler = {
+    ExceptionHandler {
```

Comment:
Should this handle `RequestValidationError` as it did before?

Reply:
Oh I see.. the new `RouteValidation` responds with `ValidationRejection` instead 👍

Reply:
yea, I figured it's more appropriate to have any validation return `ValidationRejection` instead of exception that'll be handled by exceptionHandler